libraries/AP_HAL_ChibiOS/hwdef/uav-dev-fc-um982/README.md

```diff
@@ -0,0 +1,149 @@
+- Weight: 22g with microSD card
+#### Power
+- 5.2V input via JST-GH
+#### Interfaces
+- **USB-C**
+- **Ethernet**
+  - 100 MBit/s
+- **PWM**
+  - 6 x PWM / DShot, 2 x Bidirectional-DShot capable
+- **RC Input**
+  - UART / RCININT multi protocol
+- **TELEM**
+  - 2-5 x Serial
+- **CAN**
+  - 2 x FD CAN with 120 Ohm resistor
+- **microSD**
+- **DEBUG**
+  - Tag-Connect
+
+
+## Pinout
+![top](uav-dev-fc-um982.png)
+
+#### S1 - SERIAL4 & SERIAL5 6 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | USART6_TX       | 3.3V         |
+| 3   | USART6_RX / RCIN| 3.3V         |
+| 4   | UART7_TX        | 3.3V         |
+| 5   | UART7_RX        | 3.3V         |
+| 6   | GND             | GND          |
+
+#### PWM - PWM / DShot / Telemetry - 8 Pin JST-GH
+| Pin | Signal Name           | Voltage      |
+|-----|-----------------------|--------------|
+| 1   | PWM / DShot 1 BIDIR   | 3.3V         |
+| 2   | PWM / DShot 2         | 3.3V         |
+| 3   | PWM / DShot 3 BIDIR   | 3.3V         |
+| 4   | PWM / DShot 4         | 3.3V         |
+| 5   | PWM / DShot 5         | 3.3V         |
+| 6   | PWM / DShot 6         | 3.3V         |
+| 7   | USART3_RX / TELEM     | 3.3V         |
+| 8   | GND                   | GND          |
+
+#### C1 - CAN FD 1 - 4 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | CAN_H           |              |
+| 3   | CAN_L           |              |
+| 4   | GND             | GND          |
+
+#### C2 - CAN FD 2 - 4 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | CAN_H           |              |
+| 3   | CAN_L           |              |
+| 4   | GND             | GND          |
+
+#### Debug - 6 Pin Tag-Connect
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +3.3V           | 3.3V         |
+| 2   | SWDIO           | 3.3V         |
+| 3   | NRST            | 3.3V         |
+| 4   | SWCLK           | 3.3V         |
+| 5   | GND             | GND          |
+| 6   | not connected   |              |
+
+#### S2 - SERIAL1 & GNSS - 6 Pin JST-GH
+| Pin | Signal Name     | Voltage      |
+|-----|-----------------|--------------|
+| 1   | +5V             | 5V           |
+| 2   | USART1_TX       | 3.3V         |
+| 3   | USART1_RX       | 3.3V         |
+| 4   | GNSS_PPS        | 3.3V         |
+| 5   | GNSS_EVENT      | 3.3V         |
+| 6   | GND             | GND          |
+
+#### ETH - Ethernet - 4 Pin JST-GH
+| Pin | Signal Name       |
+|-----|-------------------|
+| 1   | RX-               |
+| 2   | RX+               |
+| 3   | TX-               |
+| 4   | TX+               |
+
+## UART Mapping
+
+|Name|Function|
+|:-|:-|
+|SERIAL0|USB|
+|SERIAL1|USART1 (MAVLink)|
+|SERIAL2|UART8 (GPS1) GNSS UM982|
```

Comment:
shoudl note that UART8 is tied internally to GNSS module as is UART2
also need to note which ports are DMA capable